include/kframework/lemmas/bytes-simplification.k

```diff
@@ -24,6 +24,16 @@ module BYTES-SIMPLIFICATION [symbolic]
         orBool lengthBytes(B) ==Int lengthBytes(D)
       [simplification]
 
+    rule [signextend-word-less-pow128]:
+      ( ( (signextend ( 15 , WORD:Int )) s<Word (115792089237316195423570985008687907853099843482180094807725896704197245534208) ) ==Int 0 ) => true
+      requires WORD <Int pow128
+      [simplification, symbolic(WORD)]
```

Comment:
What number is this? We should give it a name:

![image](https://github.com/runtimeverification/evm-semantics/assets/12721972/605c5146-cf53-4155-8a1f-71bd433bdc6e)

It's 32 `F` followed by one `8` followed by 31 `0` in hex.

Reply:
@iFrostizz discovered that it is `maxUInt256 - maxUInt128 / 2`.

Reply:
This is also `not` `type(int128).max`
![image](https://github.com/runtimeverification/evm-semantics/assets/51274081/28458707-f783-4c98-8978-f779b0c1fa99)
So it's `type(int128).min`
![image](https://github.com/runtimeverification/evm-semantics/assets/51274081/b81ef4a4-44f8-439c-9867-57c27aeb8326)

Reply:
In the signed form, it's `typeint(128).min`
![image](https://github.com/runtimeverification/evm-semantics/assets/51274081/82e21500-4397-401e-8553-d137fd3693f3)